app/move/views/confirmation.njk

```diff
@@ -35,7 +35,7 @@
           name: move.profile.person.fullname | upper,
           location: location,
           date: move.date | formatDateWithDay,
-          supplier: supplierNames | oxfordJoin
+          supplier: move.supplier.name if move.supplier.name else t("supplier_fallback")
```

Comment:
I believe you can do `move.supplier.name or t("supplier_fallback")` in nunjucks. It checks for deep properties, and if they don't exist, use the fallback after the `or`.